Client/Teztez/Core/AppDelegate/AppDelegate.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AppDelegate.swift
+//  Teztez
+//
+//  Created by Almas Zainoldin on 4/18/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+    private let configurator = AppDelegateConfiguratorFactory.makeDefault()
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        setupWindow()
+        _ = configurator.application?(application, didFinishLaunchingWithOptions: launchOptions)
+        return true
+    }
+
+    private func setupWindow() {
```

Comment:
Also it'd be nice if we can finally move this setup to `StartupAppDelegateConfigurator` (the thing me & Nurlan discussed). Please try doing it